EstateManagement/Factories/ModelFactory.cs

```diff
@@ -63,7 +63,7 @@ public EstateResponse ConvertFrom(Estate estate)
         /// </summary>
         /// <param name="merchant">The merchant.</param>
         /// <returns></returns>
-        public MerchantResponse ConvertFrom(Merchant merchant)
+        public MerchantResponse ConvertFrom(Merchant merchant, MerchantBalance merchantBalance = null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/manual/stuart_ferguson1/EstateManagement/pullRequest?prid=5660966)